packages/react-native/ReactAndroid/src/main/java/com/facebook/react/bridge/JavaOnlyArray.java

```diff
@@ -156,6 +161,11 @@ public void pushInt(int value) {
     mBackingList.add(new Double(value));
   }
 
+  @Override
+  public void pushLong(long value) {
+    mBackingList.add(new Double(value));
```

Comment:
It's not ideal that the `long` API stores a double, and doesn't support values > 2^53. It is the only lossy option here.

Would it be too complicated to store as `Long` here, or to instead, use existing `double` API to make this truncation explicit?

If we are modeling JS number that's a double under the hood, and has that limitation, it might be best to model that until we can safely convert to longish.

Reply:
Hmm, this would shave off the color space, and part of the alpha, if we tried to put an android color long through it.

<img width="792" alt="image" src="https://github.com/facebook/react-native/assets/835219/c481346a-88c2-442d-823d-195da03b0515">

Reply:
Oh good catch. I think I accidentally brought that over from pushInt.